tests/12login/01threepid-and-password.pl

```diff
@@ -0,0 +1,50 @@
+my $password = "my secure password";
+
+
+test "Can login with 3pid and password using m.login.password",
+   requires => [ local_user_fixture( password => $password ), id_server_fixture() ],
+
+   check => sub {
+      my ( $user, $id_server) = @_;
+
+      my $http = $user->http;
+
+      my $medium = "email";
+      my $address = 'bob@example.com';
+
+      my $sid = $id_server->validate_identity( $medium, $address, "a client secret");
```

Comment:
If you're going to specify a client secret, you should probably actually pass it later.

A valid answer here is not to specify the client secret here.